lib/translations_checker/checker.rb

```diff
@@ -0,0 +1,93 @@
+require "translations_checker/diff"
+require "translations_checker/issues_presenter"
+require "translations_checker/terminal_colours"
+require "translations_checker/indentation"
+require "translations_checker/git_diff"
+require "translations_checker/change_checker"
+
+module TranslationsChecker
+  class Checker
+    include TerminalColours
+
+    using Indentation
+
+    def file_issues(en_diff)
+      translated_locales.flat_map { |locale| check_translation(locale, en_diff) }
+    end
+
+    def issues
```

Comment:
The same with this one, can it be private?

Reply:
Done.